README.md

```diff
@@ -256,8 +256,8 @@ We use GitHub issues in this repository to track features, bugs, and enhancement
 ## Credits and references
 
 Related projects
-  - https://github.com/cfpb/hmda-combined-documentation - ReactJS Front-end with intergration DocSearch program repository powering the [HMDA Platform](http://ffiec.cfpb.gov/)
-  - https://github.com/cfpb/hmda-platform-larft - Repo for the [Public Facing LAR formatting tool](https://ffiec.cfpb.gov/tools/lar-formatting)
-  - https://github.com/cfpb/hmda-test-files - Repo for automatically generating various different test files for HMDA Data
-  - https://github.com/cfpb/hmda-census - ETL for geographic and Census data used by the HMDA Platform
-  - https://github.com/cfpb/HMDA_Data_Science_Kit - Repo for HMDA Data science work as well as Spark codebase for [Public Facing A&D Reports](https://ffiec.cfpb.gov/data-publication/disclosure-reports/2018)
+- https://github.com/cfpb/hmda-combined-documentation - ReactJS Front-end with intergration DocSearch program repository powering the [HMDA Platform](http://ffiec.cfpb.gov/)
+- https://github.com/cfpb/hmda-platform-larft - Repo for the [Public Facing LAR formatting tool](https://ffiec.cfpb.gov/tools/lar-formatting)
+- https://github.com/cfpb/hmda-test-files - Repo for automatically generating various different test files for HMDA Data
+- https://github.com/cfpb/hmda-census - ETL for geographic and Census data used by the HMDA Platform
+- https://github.com/cfpb/HMDA_Data_Science_Kit - Repo for HMDA Data science work as well as Spark codebase for [Public Facing A&D Reports](https://ffiec.cfpb.gov/data-publication/disclosure-reports/2018)
```

Comment:
Tried reverting this but for some reason it doesn't revert here, just ignore this. No changes break anything

Reply:
Seems like all whitespace differences. I just do "git checkout foo" to restore the file from the head of the branch.